docs/developer-guide/mapstore-migration-guide.md

```diff
@@ -29,6 +29,51 @@ Please update your Node version accordingly on your develop machine or CI/CD.
 
 See the [requirements](./requirements.md#debug-build) section of the documentation for the details.
 
+The same for projects of derived `MapStoreExtensions`. Make sure to:
+
+- Update `package.json` with the new versions of webpack and other dependencies.
+- Remove and regenerate your `package-lock.json` with `npm install` after updating the Node version.
+
+Here the diff used for MapStore Extensions to update the `package.json` file (tested with Node 16):
```

Comment:
@offtherailz @tdipisa  I think we are not able to use node 20 because in this PR https://github.com/geosolutions-it/MapStore2/pull/10457 I missed the update of the createExtensionWebpackConfig method with the new hashFunction . I made a quick test adding the hashFunction equal to "xxhash64" in this [branch](https://github.com/allyoucanmap/MapStore2/tree/issue_10455_ext) and the MapStoreExtension repo seems to work with node 20.
I can open a PR with the fix so maybe we can remove the note `(tested with Node 16)` here?

Reply:
```suggestion
Here the diff used for MapStore Extensions to update the `package.json` file:
```

Reply:
everything tested and working with node 20, so I applied my changes. Please re-review